packages/web/components/Container.tsx

```diff
@@ -15,6 +15,52 @@ export const PageContainer: React.FC<FlexProps> = ({ children, ...props }) => (
   </Flex>
 );
 
+// export const ScrollContainer: React.FC<FlexProps> = ({ children, ...props }) => {
```

Comment:
If this is not used anywhere anymore - we could just delete it.

Reply:
Will remove.